docs/eks-jobs-migration.md

```diff
@@ -0,0 +1,216 @@
+# Migrating Kubernetes Jobs To The EKS Prow Build Cluster
+
+In an ongoing effort to migrate to community-owned resources, SIG K8S Infra and
+SIG Testing are working to complete the migration of jobs from the Google-owned
+internal GKE cluster to community-owned clusters.
+
+Most of jobs in the Prow Default Build Cluster should attempt to migrate to
+`cluster: eks-prow-build-cluster`. For criteria and requirements, please
+pay attention to this document.
+
+## What is EKS Prow Build Cluster?
+
+EKS Prow Build Cluster (`eks-prow-build-cluster`) is an AWS EKS based cluster
+owned by community that's used for running ProwJobs. It's supposed to be
+similar to our GCP/GKE based clusters, but there are some significant
+differences such as:
+
+- Operating system on worker nodes where we run jobs is Ubuntu 22.04
```

Comment:
Actually, it's 20.04 (EKS optimised)
<img width="507" alt="Screenshot 2023-06-30 at 09 12 39" src="https://github.com/kubernetes/test-infra/assets/9121459/bcde2eb7-0dd5-4a0c-a835-4e558e14fec1">